### DISCUSSION TITLE
`<q-select>` can i change the event fired when removing one element from `multiple` and `use-chips`

### DISCUSSION THREAD

**Discussion by Freymaurer** [quasarframework/quasar#16602] [General - Components / Directives / etc] [answered]
I want to use a `<q-select>` component with the `multiple`, and `use-chips` attribute and the option to add new values.

If a "new value" is created and the user removes this value again, i want to remove it from the list of possible selects.

I tried using the `@clear` event but it does not fire at all when i remove the selected "new value"

To clarify: In the image below i have added a new value "new value", when i now press "x" to remove it from the selected values i also want to remove it from the list of possible options (below). What event fires when i press "x"? Can i adjust it?
![image](https://github.com/quasarframework/quasar/assets/39732517/fc6f4224-9fa2-4bdb-813e-6f9529059a72)

Thanks in advance!

**Comment by pdanpdan** [comment]
Make a codepen

**Reply by Freymaurer** [reply]
How can i fire `logDeselect`, whenever i remove a single value from the list of selected items:

[code pen](https://codepen.io/freymaurer/pen/abXEZYb?editors=1010)

**Reply by pdanpdan** [reply, marked as answer]
`@remove`
https://codepen.io/pdanpdan/pen/vYbpXNW?editors=1010

**Reply by Freymaurer** [reply]
Thanks a lot! It is mentioned in the docs, i just looked at the wrong place, sorry about that!
